packages/eset_protect/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.7.0"
+  changes:
+    - description: Add support for eset notifications (https://help.eset.com/protect_admin/12.0/en-US/events-exported-to-json-format.html?admin_ntf_notifications.html). Notifications are currently not sent in json format, enhancement request already open.
```

```suggestion
    - description: Add support for eset notifications. Notifications are currently not sent in json format, enhancement request already open.
```

Can you add a proposed commit message to the PR and you can add this link over there.